app/models/experimentation/experiment.rb

```diff
@@ -7,17 +7,18 @@ class Experiment < ActiveRecord::Base
     validates :state, presence: true
     validates :experiment_type, presence: true
     validate :date_range, if: proc { |experiment| experiment.start_date_changed? || experiment.end_date_changed? }
-    validate :one_live_experiment_per_type
+    validate :one_active_experiment_per_type
 
     enum state: {
       new: "new",
       selecting: "selecting",
-      live: "live",
+      running: "running",
       complete: "complete"
-    }, _prefix: true
+    }, _suffix: true
```

Comment:
Using the suffix form here reads better:
```
experiment.running_state?
experiment.selecting_state?
```

Reply:
Yeah, that does probably read better. We use prefix on all other enums throughout the app. Maybe it reads better for those uses.